src/views/Give/ManageDonationModal.tsx

```diff
@@ -424,7 +406,7 @@ export function ManageDonationModal({
         <Grid item xs={4}>
           <Box>
             <Typography variant="h5" align="center">
-              {project ? parseFloat(totalDebt).toFixed(2) : "N/A"}
+              {project ? parseFloat(getTotalDebt()).toFixed(2) : "N/A"}
```

Comment:
Use a BigNumber and avoid `parseFloat`. Then you can use BigNumber's `toFixed` method.